user-guide/Reference/faq/DataMiner_client_applications.md

```diff
@@ -70,4 +70,6 @@ To reinstall the browser app, you need to clean the XBAP cache:
 
    1. Click *Run*.
 
+1. Try to connect using the desktop app instead. See [Installing the DataMiner Cube desktop application](xref:Installing_the_DataMiner_Cube_desktop_application)
```

Comment:
This procedure relates specifically to connecting with the browser app, so telling the user to use the desktop app instead halfway through the procedure is not helpful or user-friendly. However, I do see the use of pointing to the desktop app instead. Could you add this as a recommendation in a note at the top of the procedure?